### DISCUSSION TITLE
Set an specific inclination percentage

### DISCUSSION THREAD

**Discussion by albertoherran** [cagnulein/qdomyos-zwift#778] [Ideas]
Is there a possibility to set an fixed  inclination % in your app? My treadmill had a fixed inclination of 1% and it would be fine to be reflected in strava at the end of the workout.

**Comment by cagnulein** [comment]
which is the bluetooth name of your treadmill? i can add this directly associated to its name. i mean the name that qz shows in the top bar
thanks

**Reply by albertoherran** [reply]
It is RunnerT

thanks
